SPECS/less/less.spec

```diff
@@ -1,31 +1,28 @@
 Summary:        Text file viewer
 Name:           less
-Version:        530
-Release:        2%{?dist}
+Version:        590
+Release:        1%{?dist}
 License:        GPLv3+
```

Comment:
There is a second license in the `LICENSE` file, which is BSD. Readme confirms this should be `GPLv3+ or BSD`